### ISSUE TITLE
VSCode automatically starts up debugger/inspector in built-in terminal ever since 1.47.0 update

### ISSUE THREAD

**Issue by rajsinghUSA** [microsoft/vscode#102240]
- VSCode Version: 1.47.0
- OS Version: Windows 10 (1909) Build 18363.900

Steps to Reproduce:

A few days ago I updated my Surface Book 2 firmware (`182.2107.139.0`) through Windows Update and perhaps foolishly at the same time updating VSCode to `1.47.0`. Afterwards, the VSCode installation seemed to have disappeared (at least the `bin` directory where the executable is kept disappeared), so it was recommended to me that I reinstall VSCode, which I did. My settings and tabs all appeared to be retained. However, for one of my projects, when VSCode starts up, it gives me this output in the terminal (without a subsequent prompt which would allow me to type):
```js
Debugger listening on ws://127.0.0.1:55430/f3f20387-0605-4a39-b807-77f02bea362f
For help, see: https://nodejs.org/en/docs/inspector
```
My usual startup procedure for my project has been that I:
1. Use WSL to navigate to the project directory I want to open, and then type `code .`
2. I would then start up my server within the VSCode terminal (bash) by typing `npm run serve` (package.json has `"serve": "vue-cli-service serve"`).

However, I obviously can't do this now that there's some debugger and inspector continuously running? Here's a screenshot showing my blank `launch.json` and the terminal:

[![debugger showing instead of command prompt][1]][1]

**So I tried some more things.**

1. I ran `cp -rf testsite testfrontend` within WSL
2. and then in the copied directory I deleted `package-lock.json` and `node_modules`
3. I subsequently ran `npm install`
4. I then started VSCode by running `code .` in the directory of the copied project within WSL

5. When VSCode opened up, the terminal wasn't showing, so I showed it (Ctrl+`), and the following lines ran automatically on their own:
[![vscode terminal startup][2]][2]

5. Then I ran `npm run serve`, and the following happened (and it is stuck on this last line shown):
[![stuck][3]][3]

6. And with this message in the Debugger Console tab simultaneously:
[![error][4]][4]

7. Furthermore, if I close out of VSCode at this point, and reopen it again with `code .` from `testfrontend` (which is the newly `cp`'d directory I created in _Step 1_) in WSL, then I am back to square one with VSCode's terminal automatically showing this agani as soon as VSCode starts up:
```js
Debugger listening on ws://127.0.0.1:55430/f3f20387-0605-4a39-b807-77f02bea362f
For help, see: https://nodejs.org/en/docs/inspector
```

----------

I should mention that I **don't get this error or any of the other aforementioned issues** in the following two scenarios:
1. when I run `npm run serve` directly from WSL
2. I also don't get the aforementioned issues if I click the green `WSL: Ubuntu` button at the bottom left and then choose `Remote-WSL: New Window` and then run `npm run serve`:
[![new window][5]][5]

Can someone please tell me how to fix this? I have no idea what has gone wrong, but it only seems to happen with this particular project.

  [1]: https://i.stack.imgur.com/c9Hs5.png
  [2]: https://i.stack.imgur.com/hJJ49.png
  [3]: https://i.stack.imgur.com/sBLsN.png
  [4]: https://i.stack.imgur.com/2WDDh.png
  [5]: https://i.stack.imgur.com/lMgU7.png

_Does this issue occur when all extensions are disabled?: n/a because I am using WSL, which requires extensions_

**Comment by connor4312** [comment]
This is the same as https://github.com/microsoft/vscode/issues/102057

You can fix this for the moment by setting `debug.node.autoAttach` to disabled, then reloading the window / reopening the terminal.

/duplicate

**Comment by rajsinghUSA** [comment]
I ended up starting VSCode with `code --disable-extensions .` from my WSL, and just ignoring all the error messages and shutting down VSCode. Afterwards, when I started VSCode normally from the project directory via `code .`, everything went back to working as normal, where my `npm run myscript` and everything else worked exactly as expected.

Though I'm unclear how `debug.node.autoAttach` would provide a solution for the error message I had shown, I anyway didn't get to try your solution this time, but I'll keep it in mind if something similarly screwy happens in the future.

Thanks
